chain/gen/genesis/genesis.go

```diff
@@ -434,10 +434,24 @@ func MakeGenesisBlock(ctx context.Context, bs bstore.Blockstore, sys vm.SyscallB
 		VRFProof: []byte("vrf proof0000000vrf proof0000000"),
 	}
 
+	filecoinGenesisCid, err := cid.Decode("bafyreiaqpwbbyjo4a42saasj36kkrpv4tsherf2e7bvezkert2a7dhonoi")
```

Comment:
1. We might want to use the multihash as stamped into eth.
2. We should _definitely_ check that this equals `gblock.Cid()`.